doc/api/v8.md

```diff
@@ -390,6 +390,82 @@ The API is a no-op if `--heapsnapshot-near-heap-limit` is already set from the
 command line or the API is called more than once. `limit` must be a positive
 integer. See [`--heapsnapshot-near-heap-limit`][] for more information.
 
+## `v8.collectGCProfile(options)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object}
+  * `duration` {number} how long you want to collect the data.
```

Comment:
This should specify the units. I assume milliseconds?

Reply:
Also, is it possible to make this cancelable with an AbortSignal?

Reply:
Yes, i have update the code to support `AbortSignal` .